cmd/nvidia_gpu/nvidia_gpu.go

```diff
@@ -151,6 +161,29 @@ func main() {
 			return
 		}
 		defer hc.Stop()
+
+	}
+
+	if *enableFlockWait {
+		kubeClient, err := util.BuildKubeClient()
+		if err != nil {
+			glog.Infof("Failed to build kube client: %v", err)
+			return
+		}
+		nodeName, err := util.GetEnv(nodeNameEnv)
```

Comment:
This would require a change in DaemonSet yaml, right? should we getHostname if failed to get this env var?